lib/datapacksutils.js

```diff
@@ -30,6 +31,12 @@ var DataPacksUtils = module.exports = function(vlocity) {
     this.fieldLabels = {};
     this.objectLabels = {};
     this.fieldImportance = {};
+    const ignoreFilePath = path.resolve(process.cwd(), '.vlocityignore');
```

Comment:
Actually instead of loading it here, please add a function to load it before it is being used, and it should load from the `jobInfo.projectPath`. This can go into the datapacksjob.js as part of initializing the job in `intializeJobInfo()`